flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/functions/hive/HiveGenericUDTF.java

```diff
@@ -157,4 +165,16 @@ public TypeInformation getResultType() {
     public void close() throws Exception {
         function.close();
     }
+
+    public static StandardStructObjectInspector getStandardStructObjectInspector(
+            ObjectInspector[] argumentInspectors) {
+        List<String> structFieldNames = new ArrayList<>();
+        for (int i = 0; i < argumentInspectors.length; i++) {
+            structFieldNames.add(String.valueOf(i));
```

Comment:
It seems like it's ok to set a arbitrary value here for we will still set colum name in function GenericUDTF.initialize().

But could you like to add some comments here for it's confused.

Reply:
done